templates/user.html

```diff
@@ -15,21 +15,25 @@
   <meta property="og:description" content="See all your Open Source contributions in one place."/>
   <!-- Open Graph End -->
 
+  {{if (ne .ResponseType "iframe")}}
   <!-- Twitter Card Start -->
```

Comment:
@karanjthakkar you can split the template into parts, put statistics and pr list into a separate file and include it in two templates: the usual one and for iframe.

Have you ever thought about using JavaScript to display information, for example React?